quill-sql/src/main/scala/io/getquill/source/sql/idiom/MySQLDialect.scala

```diff
@@ -16,5 +17,16 @@ trait MySQLDialect
       case BinaryOperation(a, StringOperator.`+`, b) => s"CONCAT(${a.show}, ${b.show})"
       case other                                     => super.operationShow.show(other)
     }
+
+  override implicit def orderByCriteriaShow(implicit strategy: NamingStrategy): Show[OrderByCriteria] = new Show[OrderByCriteria] {
+    def show(criteria: OrderByCriteria) =
+      criteria match {
+        case OrderByCriteria(prop, AscNullsFirst)  => s"${prop.show} ASC"
+        case OrderByCriteria(prop, DescNullsFirst) => s"-${prop.show} ASC"
```

Comment:
I see, it's not consistent across different versions of mysql then. I'm changing it to use `ISNULL`.